tests/phpunit/tests/theme/wpThemeJson.php

```diff
@@ -3595,6 +3595,66 @@ public function test_get_styles_for_block_with_content_width() {
 		$this->assertSame( $expected, $root_rules . $style_rules );
 	}
 
+	/**
+	 * @ticket 57583
+	 */
+	public function test_get_styles_for_block_with_style_variations() {
+		$theme_json = new WP_Theme_JSON(
+			array(
+				'version' => 2,
+				'styles'  => array(
+					'blocks' => array(
+						'core/quote' => array(
+							'variations' => array(
+								'plain' => array(
+									'color'                => array(
+										'background' => 'hotpink',
+									),
+									'unregisteredProperty' => 'value',
+								),
+							),
+						),
+					),
+				),
+			),
```

Comment:
```suggestion
			)
```

Reply:
Trailing commas in parameter list are allowed from 7.3 and above, so that may be the reason why the tests fail for anything below 7.2. I didn't catch that when I prepared the test locally, sorry about all the troubles in integrating it here!

Reply:
Let me take a look. I'll push changes to fix the tests.